Client/Components/TrackedItemComponent.razor

```diff
@@ -41,13 +43,36 @@
 
 <p>@ItemToTrack.PastOccurrences.Count instances tracked</p>
 <p>Last Tracked: @LastTracked</p>
-@if (ItemToTrack.Targets.Any())
+@if (ItemToTrack.Targets.Any(t => t.Qty > 0 && t.Frequency > TimeSpan.Zero))
 {
     <p>
         Next Due: @NextDue @if (!string.IsNullOrEmpty(NextSpaced))
         {
             <span> (@NextSpaced)</span>
         }
+        
```

Comment:
We don't want to show this block here otherwise the main page will be too busy. Only show the nofication options when Editing a TrackedItem